pkg/tasks/upgrade_follower.go

```diff
@@ -51,13 +51,17 @@ func upgradeFollowerExecutor(s *state.State, node *kubeoneapi.HostConfig, conn s
 		return errors.Wrap(err, "failed to drain follower control plane node")
 	}
 
+	if err := setupProxy(logger, s); err != nil {
+		return err
+	}
```

Comment:
The problem with this approach is that it's going to restart the container runtime. While it's okay-ish for containerd, it might not be okay for Docker because it's going to restart all containers. Do we have some other way around? Can we only update the environment file here?

Reply:
OK, I've changed a bit whole concept

Reply:
On the other hand, by the time when docker is restarted the node **IS DRAINED AND SCHEDULING IS DISABLED**, so any remaining pods are safe to restart.